lutris/gui/views/base.py

```diff
@@ -28,35 +28,57 @@ def popup_contextual_menu(self, view, event):
         """Contextual menu."""
         if event.button != Gdk.BUTTON_SECONDARY:
             return
-        view.current_path = view.get_path_at_pos(event.x, event.y)
-        if view.current_path:
-            view.select()
-            model = self.get_model()
-            _iter = model.get_iter(view.current_path[0])
-            if not _iter:
-                return
-            col_id = str(model.get_value(_iter, COL_ID))
-
-            game_actions = self.get_game_actions(col_id)
+        game_ids = []
+        if self.view_type == "grid":
+            selection = self.get_selected_items()
+        else:
+            selection = self.get_selection().get_selected_rows()
+            selection = selection[1]
+        if len(selection) < 2:
+            view.current_path = view.get_path_at_pos(event.x, event.y)
```

Comment:
I think we should handle the single selection and multi-selection with the same code. I don't think we need get_path_at_pos(), not unless there is no selection- and maybe we should not just have a context menu in that case. Having a selection of one, but then opening the menu for *some other game* would be confusing.

I think we could just use the 'else' block code only.